selfdrive/car/ford/fordcan.py

```diff
@@ -136,9 +136,6 @@ def create_acc_msg(packer, CAN: CanBus, long_active: bool, gas: float, accel: fl
     "AccPrpl_A_Pred": gas,                            # Acceleration request: [-5|5.23] m/s^2
     "AccResumEnbl_B_Rq": 1 if long_active else 0,
     "AccVeh_V_Trg": v_ego_kph,                        # Target speed: [0|255] km/h
-    # Precharging and actuating the actual brake signals are typically only done ine BlueCruise when approaching INACTIVE_GAS (-5)
```

Comment:
oh this top one is good. I just meant the other two lines that reference old code. comments should just explain the current state of things